docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/net/index.md

```diff
@@ -49,6 +49,7 @@ The final step is to configure the exporter endpoint, service and application na
 * `OTEL_EXPORTER_OTLP_ENDPOINT=http://collection-sumologic-otelagent.sumologic:4318` - environment variable configures the endpoint where telemetry data will be sent. The value of the variable points to the default Sumologic Kubernetes Collector.
 * `OTEL_SERVICE_NAME=SERVICE_NAME` - configure the service name. Ensure the string value represents its business logic, such as "FinanceServiceCall". This will appear as a tracing service name in Sumo Logic.
 * `OTEL_RESOURCE_ATTRIBUTES=application=APPLICATION_NAME` - configure the application name. This will appear as a tracing application name in Sumo Logic. Additional attributes can be added here as comma separated key=value pairs.
+* {@import ../../../../../reuse/otel-deployment-environment-tag.md}
```

Comment:
Please put it as a part of `OTEL_RESOURCE_ATTRIBUTES`. It's not separated environment variable.